src/components/Anchor/Anchor.js

```diff
@@ -29,6 +29,19 @@ const inferType = children => {
   return 'wrap';
 };
 
+const matchReactRouter = (to, match, exact) => {
+  let active;
+  if (!exact) {
```

Comment:
This could probably be refactored to avoid the need for declaring a variable, but that's mostly a stylistic issue, which is always subjective.